src/platforms/common/enriching-events/viewhierarchy.mdx

```diff
@@ -13,6 +14,8 @@ notSupported:
   - dotnet.xamarin
 ---
 
+<Include name="alpha-note.mdx" />
```

Comment:
L: Should this be displayed for all platforms?

Reply:
Yes, every platform is alpha, it was missing.